rpc/glfs-operations.c

```diff
@@ -36,7 +38,12 @@ glusterBlockVolumeInit(char *volume, int *errCode, char **errMsg)
     return NULL;
   }
 
-  ret = glfs_set_volfile_server(glfs, "tcp", "localhost", 24007);
+  port = gbConf.glusterdPort;
+  hostname = gbConf.glusterdHostname;
+
+  ret = glfs_set_volfile_server(glfs, "tcp",
+                                ((hostname)? hostname: "localhost"),
+                                ((port)? port: 24007));
```

Comment:
Are the "()" needed for hostname and port here ?

Reply:
Maybe we should init the default value in:
`
20 struct gbConf gbConf = {LRU_COUNT_DEF, GB_LOG_INFO, GB_LOGDIR, '\0', '\0', '\0', '\0', '\0'};
`

Reply:
Done